lib/synapse_pay_rest/http_client.rb

```diff
@@ -19,15 +20,14 @@ class HTTPClient
     # @param log_to [String] (optional) file path to log to file (logging must be true)
     # @param proxy_url [String] (optional) proxy url which is used to proxy outbound requests
     def initialize(base_url:, client_id:, fingerprint:, ip_address:,
-                   client_secret:, development_mode:, vgs_url:, **options)
+                   client_secret:, development_mode:, **options)
       log_to         = options[:log_to] || 'stdout'
       RestClient.log = log_to if options[:logging]
       @logging       = options[:logging]
       @development_mode = development_mode
 
-      @vgs_url = vgs_url
-      @vgs_sandbox_pem = options[:sandbox_pem]
-      @vgs_live_pem = options[:live_pem]
+      @vgs_url = options[:vgs_url]
+      @vgs_pem = options[:vgs_pem]
```

Comment:
With this change, the logic to decide what vgs_url and vgs_pem to use per env must be handled by the app using this gem, that way this gem just uses whatever values are passed.